src/styles/main.scss

```
@@ -1,3 +1,36 @@
+$square-size: 100px;
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
```

Move font family to the html selector
```suggestion
  font-family: Arial, Helvetica, sans-serif;
```